rust/operator-binary/src/backend/tls/ca.rs

```diff
@@ -34,6 +34,7 @@ use tracing::{info, info_span, warn};
 
 use crate::{
     backend::SecretBackendError,
+    crd::TlsKeyGeneration,
```

Comment:
note: As mentioned above, this might not be the best name. One could use `CertificateSigningKeyGeneration`.

Reply:
Yeah its not perfect, but as stated before will not stay like this forever. Will be properly reworked if we switch to the op-rs libs.

Reply:
> Just FYI the field is used to control both - the ca cert key and the pod cert key. I don't know if the name is still valid ;)

In my opinion it is still valid. Both certificates are signed. But we can maybe shorten it to `CertificateKeyGeneration`.

Reply:
Used `CertificateKeyGeneration` in https://github.com/stackabletech/secret-operator/pull/506/commits/077d2bb33b31eb91c517c391138d08be87f49961